src/jwplayer/players/DesktopArticleVideoPlayer/DesktopArticleVideoPlayer.tsx

```diff
@@ -62,6 +66,9 @@ export const DesktopArticleVideoPlayerContent: React.FC<DesktopArticleVideoPlaye
 
 	const onPlayerInstanceReady = (playerInstance) => {
 		articlePlayerOnReady(videoDetails, playerInstance);
+		communicationService.on('[JWPlayer] Player Ready', async () => {
```

Comment:
Is "player instance ready" a different stage compared to "player ready"? In other words, is there a specific reason to listen for this event within the "onPlayerInstanceReady" callback? 🤔

Reply:
It seems to me to be a good moment to call it. We can change it if you have other ideas.

Reply:
The moment is perfect, but is there a specific reason we need to involve the communication service in this case? Why even bother if the player instance is ready?